README.md

```diff
@@ -5,52 +5,68 @@
+instead, for when you create pull requests against the content):
 
     git clone https://github.com/mdn/content.git mdn/content
 
-Now, you just need to note where that folder is before you can start Yari.
+Take a note of where you've clone that repo before moving on.
 
 To run Yari locally, you'll first need to install [git](https://git-scm.com/),
-[Node.js](https://nodejs.org) (>= 12.0.0) and
+[Node.js](https://nodejs.org) (>= 12.0.0), and
 [Yarn 1](https://classic.yarnpkg.com/en/docs/install).
-After that, run these commands in your bash:
+
+After that, run these commands in your terminal, bearing in mind that you'll
+probably want to fork the repo first and do the `git clone` with _your_ fork
+instead of the `mdn` one:
 
     git clone https://github.com/mdn/yari.git
     cd yari
-    yarn
+
+Next run the `yarn` command to fetch all the dependencies and install the app
+locally:
+
+Now run the following command to create a `.env` file inside your `yari` repo
+root and set the `CONTENT_ROOT` environment variable equal to the path to the
+`content` repo. This is so the Yari app can find the content it needs to render.
+You'll need to replace `/path/to/mdn/content/files` with the path to the
+`/files` folder inside your clone of the `content` repo:
+
     echo CONTENT_ROOT=/path/to/mdn/content/files >> .env
+
+At this point, you can get started. Run the following lines to compile required
+files, start the Yari web server running, and open it in your browser:
+
     yarn dev
     open http://localhost:3000
 
-Make sure you point to the `/files` folder inside your clone of the content
-repo.
+If you prefer you can use `yarn start`, which will re-use any previously
+compiled files; this is "riskier" but faster. `yarn dev` always ensures that
+everything is up-to-date.
 
-If you prefer, you can fork the repo first and do the `git clone` with
-_your_ fork instead of the `mdn` one.
-
-The `yarn dev` command will compile and prepare certain files. This always
-takes a little extra time. If you prefer you can use `yarn start` which
-will re-use any previously compiled files which is "riskier" but faster.
-The `yarn start` command will also start a server which doesn't automatically
-reload when its source code files change, so use with caution.
+The `yarn start` command also starts a server with slightly different behavior —
+it doesn't automatically reload when its source code files change,
+so use with caution.
 
 See also our [reviewing guide](docs/REVIEWING.md) for information on how to
 review Yari changes.
 
 ### How to stay up-to-date
 
-Periodically, the code and the content changes. Make sure you're staying
-up-to-date with these commands:
+Periodically, the code and the content changes. Make sure you stay
+up-to-date with something along the following lines (replace `yari-origin`
+with whatever you called the remote location of the original yari repo):
 
-    git pull origin master
+    git fetch yari-origin
```

Comment:
I don't think we should encourage this. `git pull` is much more convenient. And it's just one line too. 
So instead of `git fetch ...` and `git rebase ...` I think we should just say:

```
git pull yari-origin main
```
which is the same as:
```
git fetch yari-origin main
git merge yari-origin/main
```

Rebase is scarier and more error-prone that `merge` in my opinion.

Reply:
Personally, I use `git pull origin --rebase` :-)
It's only getting scary when things go wrong …

Reply:
I'm happy to go with Peter's advice here. This is just an example anyway — people will update their fork in whichever way they want.

This is coming in the next push. I'll also update any stray instances of `master` with `main`